gtfs-realtime/README.md

```diff
@@ -3,5 +3,6 @@ This directory contains GTFS Realtime Specification and documentation.
 ### Quick links
 - [GTFS Realtime protocol buffer definition](proto/gtfs-realtime.proto)
 - [Documentation](spec/en)
+- [Documentación en Español](spec/es)
```

Comment:
It would probably make sense to split languages out as sub-bullets, like:

- Documentation
    * [English](spec/en)
    * [Español](spec/es)